Prototypes/SprintOneReadMe.txt

```diff
@@ -0,0 +1,18 @@
+Sprint 1 Progress: Hayden Karl
```

Comment:
@haydenkarl22 We don't want to include 'sprint' specific files. I suggest pasting the contents of this ReadMe file into the pull request comment and removing this SprintOneReadMe.txt file. Also, all commits you make are tied to your GitHub profile, so you don't need to include your name on them (we know who made the commits and who created the PRs)